packages/contracts-bedrock/test/L1/OptimismPortal.t.sol

```diff
@@ -201,7 +201,6 @@ contract OptimismPortal_Test is CommonTest {
     }
 
     /// @dev Tests that `depositTransaction` succeeds when msg.sender == tx.origin and non-custom gas is used.
-    /// forge-config: ciheavy.fuzz.runs = 8192
```

Comment:
when should a test have this? Is there a way to automatically know when a new test is written that needs it?

Reply:
oh... I read this as an addition, not deletion.